message_definitions/v1.0/common.xml

```diff
@@ -3507,11 +3516,11 @@
       <field type="uint8_t" name="rtk_health">GPS-specific health report for RTK data.</field>
       <field type="uint8_t" name="rtk_rate" units="Hz">Rate of baseline messages being received by GPS, in HZ</field>
       <field type="uint8_t" name="nsats">Current number of sats used for RTK calculation.</field>
-      <field type="uint8_t" name="baseline_coords_type">Coordinate system of baseline. 0 == ECEF, 1 == NED</field>
+      <field type="uint8_t" name="baseline_coords_type" enum="RTK_BASELINE_COORDINATE_SYSTEM">Coordinate system of baseline</field>
       <field type="int32_t" name="baseline_a_mm" units="mm">Current baseline in ECEF x or NED north component in mm.</field>
       <field type="int32_t" name="baseline_b_mm" units="mm">Current baseline in ECEF y or NED east component in mm.</field>
       <field type="int32_t" name="baseline_c_mm" units="mm">Current baseline in ECEF z or NED down component in mm.</field>
-      <field type="uint32_t" name="accuracy">Current estimate of baseline accuracy.</field>
+      <field type="uint32_t" name="accuracy" unit="c%">Current estimate of baseline accuracy</field>
```

Comment:
How accurate is 100%? When talking about equipment that measures absolute positions anywhere from sub centimeter to 100+ levels percentages become meaningless. Especially given that you are trying to reduce this to a limited range. This proposal wastes > half the field size, and at the end of the day isn't a helpful metric to the operator.